tests/test_dense_customized.py

```diff
@@ -42,10 +43,51 @@ def test_dense_input_xor_customized():
     ax.contourf(xx, yy, Z, alpha=.3, cmap=plt.cm.Paired)
     ax.axis('off')
     fig.savefig("test_model_xor_customized.png")
-    dg = DenseGraph(model, input_color='#FF0000', inner_color='#00FF00', output_color='#FFFF00',
-                    edge_clr='#0000FF', input_shape='circle', inner_shape='diamond', output_shape='polygon')
+    dg = DenseGraph(model)
+    dg.customize_graph(input_node_color='#FF0000', inner_node_color='#00FF00', output_node_color='#FFFF00',
+                       edge_clr='#0000FF', input_node_shape='circle', inner_node_shape='oval',
+                       output_node_shape='pentagon')
     dg.render(X, filename='test_input_xor_customized')
 
+    # test: set nodes manually
+    the_graph = dg.get_graph()
+    for l in range(len(model.layers)):
+        layer = model.layers[l]
+
+        for n in range(0, layer.input_shape[1]):
+            if l == 0:
+                the_graph.add_node(
+                    str(l) + str(n),
+                    shape=get_random_shape(),
+                    color=get_random_color(),
+                    label=''
+                )
+            else:
+                the_graph.add_node(
+                    str(l) + str(n),
+                    shape=get_random_shape(),
+                    color=get_random_color(),
+                    label=''
+                )
+
+            for h in range(0, layer.output_shape[1]):
+                if l == len(model.layers) - 1:
+                    the_graph.add_node(
+                        str(l + 1) + str(h),
+                        shape=get_random_shape(),
+                        color=get_random_color(),
+                        label=''
+                    )
+                the_graph.add_edge(
+                    str(l) + str(n),
+                    str(l + 1) + str(h),
+                    color=get_random_color()
+                )
```

Comment:
You shouldn't be rebuilding the graph here - instead iterate over `the_graph` and customize it. That aside, IMO this is exactly what we want users to do when customizing their graph. I don't think we need the `customize` function if we have good documentation on how to easily customize the graph this way.